regolith/builders/recentcollabsbuilder.py

```diff
@@ -104,23 +109,31 @@ def latex(self):
                             print(
                                 "WARNING: {} missing from institutions".format(
                                     pinst))
-                ppl_names = [(person["name"], i) for
+                ppl_names = [('A', person["name"], i, '', '') for
                              person, i in zip(people, institutions) if
                              person]
                 #                print(set([person["name"] for person in people if person]))
-                print(set([person for person in ppl_names]))
-            emp = p.get("employment", [{"organization": "missing",
+        #print(set([person for person in ppl_names]))
+                emp = p.get("employment", [{"organization": "missing",
                                         "begin_year": 2019}])
-            emp.sort(key=ene_date_key, reverse=True)
-            self.render(
-                "recentcollabs.csv",
-                p["_id"] + ".csv",
-                p=p,
-                title=p.get("name", ""),
-                pubs=pubs,
-                employment=emp,
-                collabs=my_collabs
-            )
+                emp.sort(key=ene_date_key, reverse=True)
+                people_df = pd.DataFrame(ppl_names)
+                out_folder = "_build/recent-collabs/"
+                out_file = 'recent_collaborators.csv'
+                people_df.to_csv(out_folder + out_file, index=False)
+
+                excel_df = pd.read_excel('../templates/coa_template.xlsx')
+                head_part = excel_df.iloc[:49, :].copy()
+                tail_part = excel_df.iloc[49:, :].copy()
+                print(head_part.shape)
+                print(tail_part.shape)
+                print(people_df.shape)
+                concat_df = pd.concat([head_part, people_df, tail_part], ignore_index=True)
+                concat_df.to_excel(out_folder + 'co.xslx')
+                break
```

Comment:
Yeah I removed it. The `break` was added so that if it finds one instance of `p["_id"] == "sbillinge"` it'll stop. Earlier the code produced many dataframes and we thought this was due to the `for p in self.gtx["people"]:` loop. But I checked again and it produced only 1 dataframe even w/o the `break` so I guess it was some error in the code level earlier.